be/src/storage/storage_engine.h

```diff
@@ -283,7 +283,7 @@ class StorageEngine {
     // map<rowset_id(str), RowsetSharedPtr>, if we use RowsetId as the key, we need custom hash func
     std::unordered_map<std::string, RowsetSharedPtr> _unused_rowsets;
 
-    bool _stop_bg_worker = false;
+    bool _bg_worker_stopped = false;
```

Comment:
Please use `std::atomic<bool>`, and use `std::memory_order_release` for writes and `std::memory_order_consume` for reads.